security-proxy/src/test/java/org/georchestra/security/ProxyTest.java

```diff
@@ -16,9 +17,10 @@
 import java.lang.reflect.Method;
 import java.net.URI;
 import java.net.URL;
+import java.util.List;
 import java.util.Map;
 
-import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.*;
```

Comment:
My ide is configured not to use "*" in import. Can be discussed.

Reply:
I don't like'em either, looks like I just forgot to hit CTRL+O or was afraid to cause too much mangling on the imports

Reply:
resolved.